docs/tools/terraform/reference/cookbook/postgres-clickhouse-integration-recipe.rst

```diff
@@ -0,0 +1,180 @@
+Aiven for PostgreSQL® as a source for Aiven for ClickHouse®
+===========================================================
+
+You can use a PostgreSQL database as a data source and Aiven for ClickHouse - to read, transform, and execute jobs using data from the PostgreSQL server. For this purpose, you need to integrate Aiven for PostgreSQL® with Aiven for ClickHouse®. Check out this article to learn how to connect these services using `Aiven Terraform Provider <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_.
+
+Let's cook!
+-----------
+
+Imagine that you've been collecting IoT measurements from thousands of sensors and storing them in ClickHouse table ``iot_measurements``. Now, you'd like to enrich your metrics by adding the sensor's location measurement so that you can filter the metrics by the city name. The sensor's location data is available in the ``sensors_dim`` database in PostgreSQL.
```

Comment:
@dewan-ahmed is correct that the introduction should read "database" instead of "table" for consistency and correctness

```suggestion
Imagine that you've been collecting IoT measurements from thousands of sensors and storing them in ClickHouse database ``iot_measurements``. Now, you'd like to enrich your metrics by adding the sensor's location measurement so that you can filter the metrics by the city name. The sensor's location data is available in the ``sensors_dim`` database in PostgreSQL.
```